modules/creating-an-infra-node.adoc

```diff
@@ -13,16 +13,20 @@
 See Creating infrastructure machine sets for installer-provisioned infrastructure environments or for any cluster where the control plane nodes are managed by the machine API.
 ====
 
-Requirements of the cluster dictate that infrastructure, also called `infra` nodes, be provisioned. The installer only provides provisions for control plane and worker nodes. Worker nodes can be designated as infrastructure nodes or application, also called `app`, nodes through labeling.
+Requirements of the cluster dictate that infrastructure, also called `infra` nodes, be provisioned. The installation program provisions only control plane and worker nodes. Worker nodes can be designated as infrastructure nodes through labeling. You can then use taints and tolerations to move appropriate workloads to the infra nodes. See "Moving resources to infrastructure machine sets".
```

Comment:
```suggestion
Requirements of the cluster dictate that infrastructure (infra) nodes, be provisioned. The installation program provisions only control plane and worker nodes. Worker nodes can be designated as infra nodes through labeling. You can then use taints and tolerations to move appropriate workloads to the infra nodes. For more information, see "Moving resources to infrastructure machine sets".
```
In the docs, it look like most instances of "infra node" do not place the term "infra" in backticks. Within this PR, I see the phrase both with "infra" in backticks and without. I'm not sure if one formatting style is more accurate than the other, but it would be good to be consistent.